docs/en/query_language/functions/rounding_functions.md

```diff
@@ -78,6 +78,92 @@ round(3.55, 1) = 3.6
 round(3.65, 1) = 3.6
 ```
 
+**See Also** 
+
+- [roundBankers](#roundbankers)
+
+## roundBankers(x[, N]) {#roundbankers}
+
+Banker's rounding is a method of rounding fractional numbers, which differs from the usual arithmetic rounding in cases when the rounding number is a half between two numbers, it is rounded to the nearest even number. E.g. 3.5 rounds up to 4, 2.5 rounds down to 2.
```

Comment:
There is no any *usual arithmetic rounding*  in the world :-) https://en.wikipedia.org/wiki/Rounding. Remove the words about *usual* and just describe the method of rounding itself.